metricbeat/module/iis/webserver/manifest.yml

```diff
@@ -0,0 +1,123 @@
+default: true
+input:
+  module: windows
+  metricset: perfmon
+  defaults:
+    perfmon.group_measurements_by_instance: true
+    perfmon.ignore_non_existent_counters: true
+    perfmon.group_all_counter: "worker_process_count"
+    perfmon.counters:
+    #network
+    - instance_label: ''
+      measurement_label: network.total_bytes_received
+      query: '\Web Service(_Total)\Total Bytes Received'
+    - instance_label: ''
+      measurement_label: network.total_bytes_sent
+      query: '\Web Service(_Total)\Total Bytes Sent'
+    - instance_label: ''
+      measurement_label: network.bytes_sent_per_sec
+      query: '\Web Service(_Total)\Bytes Sent/sec'
+    - instance_label: ''
+      measurement_label: network.bytes_received_per_sec
+      query: '\Web Service(_Total)\Bytes Received/sec'
+    - instance_label: ''
+      measurement_label: network.current_connections
+      query: 'Web Service(_Total)\Current Connections'
```

Comment:
Would be great to also have 
`Web Service(*)Current Anonymous Users`
`Web Service(*)Current NonAnonymous Users`
`Web Service(*)Total Anonymous Users`
`Web Service(*)Anonymous Users/sec`
`Web Service(*)Total NonAnonymous Users`